build/install-node.sh

```diff
@@ -11,7 +11,8 @@
 # catch Errors
 set -euo pipefail
 
-NODE_VERSION="16"
+# Updated install-node.sh due to the following issue, fixed in a more specific LTS version of 16 (v16.20.2 - gallium):
+NODE_VERSION="v16.20.2"
```

Comment:
@syedahmedhaidershah for the same reasons we discussed using the `~13.1` for cypress we want to ensure we are using the latest version of node. Can you do further testing to understand why setting it to 16 did not work? I would recommend running the base image directly and then manually running these commands. It's possible that NVM needs to be updated as well.